src/definitions/modules/progress.js

```diff
@@ -423,11 +423,19 @@ $.fn.progress = function(parameters) {
             else {
               var firstNonZeroIndex = -1;
               var lastNonZeroIndex = -1;
+              var percentSum = 0;
+              var barCounts = $bars.length;
+              var isMultiple = barCounts > 1;
               var percents = values.map(function(value, index) {
+                percentSum += value;
```

Comment:
..then this line can be removed (because in the end it only counts if **all** values are zero

Reply:
Fixed in https://github.com/fomantic/Fomantic-UI/pull/625/commits/730f0c12011d18b14ae760bcd443ec46571b1f09